airflow/providers/microsoft/azure/hooks/asb.py

```diff
@@ -114,6 +119,93 @@ def delete_queue(self, queue_name: str) -> None:
         with self.get_conn() as service_mgmt_conn:
             service_mgmt_conn.delete_queue(queue_name)
 
+    def create_subscription(
```

Comment:
I suggest to just use the underlying client method rather than adding this one

Reply:
You are suggesting to call the create_subscription client method in operator ? if yes how will I get the connection ? should I call the `get_conn()` method in operator ?

Reply:
I agree with your point instead adding method in hook its better to add in operator, it will be lighter weight. But my only doubt can I use client methods  in operator call the `get_conn()` method in operator, no where I can seeing in the repo using client method directly in operator

Reply:
You can do that, it's no problem at all. It is done for sure.

Reply:
will do that and raise the PR

Reply:
@dstandish I have addressed your suggestion, can you please review and approve it